<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameImageFolderClass</class>
 <widget class="QDialog" name="RenameImageFolderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RenameImageFolder</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>renameImageFolder</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Name Filter:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Source Folder:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sourceFolder">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>240</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nameFilter">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>240</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browseSourceFolder">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_execute">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>180</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>FileType:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_fileType">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>240</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_addNum">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Num</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_addTime">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Time</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_undo">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>180</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Dest Filter:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_destFilter">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>240</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_folderOnly">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>110</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Folder Only</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
